tests/RobotFramework/tests/configuration/lock_file.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Resource    ../../resources/common.resource
+Library    ThinEdgeIO
+
+Test Tags    theme:configuration
+Suite Setup    Setup
+Test Teardown   Get Logs
+
+*** Test Cases ***
+
+Check lock file existence in default folder
+    [Documentation]    When deploying thin-edge components in single-process containers, 
+    ...    the lock file mechanism used by some components (e.g. tedge-agent, tedge-mapper) 
+    ...    do not makes sense as the container filesystem is isolated. 
+    ...    Having the lock file system just adds unnecessary dependencies on the /run/lock folder.
+    File Should Exist    /run/lock/tedge-agent.lock
+    File Should Exist    /run/lock/tedge-mapper-c8y.lock
+
+Check starting same service twice
+    [Documentation]    This step is checking if same service can be started twice, 
+    ...    expected is that this should not be the case
+    Execute Command    sudo runuser -u tedge tedge-agent    exp_exit_code=!0
+    Execute Command    sudo runuser -u tedge tedge-mapper-c8y    exp_exit_code=!0
```

Comment:
I think this execute command returns non-zero because`tedge-mapper-c8y` doest not exist. 

```suggestion
    Execute Command    sudo runuser -u tedge tedge-mapper c8y    exp_exit_code=!0
```